Gems/Meshlets/ASV_GPU_and_CPU_Demo/Readme.txt

```diff
@@ -163,18 +179,23 @@ is not the case, simply replace with the directory name of your active project.
                 Meshlets
             )
 
-4. Copy the shader debug files (DebugShaderPBR_ForwardPass.*) to the shader directory:
+4. Add a build dependency on the meshlets gem - AtomSampleViewer/Gem/Code/CMakeLists.txt
+        ly_add_target(
+            NAME AtomSampleViewer.Private.Static STATIC
+            ...
+            BUILD_DEPENDENCIES
+                PUBLIC
+                    Gem::Meshlets.Static
+
+5. Copy the shader debug files (DebugShaderPBR_ForwardPass.*) to the material types directory:
     [o3de]\AtomSampleViewer\Materials\Types\*
 
-5. Add the material file 'debugshadermaterial_01.material' to the material directory:
-    [o3de]\AtomSampleViewer\Materials\DebugShaderMaterial_01.material
+6. Copy the material type file 'DebugShaderPBR.materialtype' to the material types directory:
+    [o3de]\AtomSampleViewer\Materials\DebugShaderPBR.materialtype
     
-6. Complie meshoptimizer to a library, copy the compiled library (you can compile and place it under Gems\Meshlets\External\Lib) 
-    and add it to your Meshlets.static project.
-    
-Remark: at this point there is no need to copy the MainPipeline.pass that was provided 
-    as an example, since the meshlets parent pass is being injected into the pipeline by the 
-    MeshletsFeatureProcessor.
+7. Copy the material file 'debugshadermaterial_01.material' to the material directory:
```

Comment:
I actually suggest now not doing it.  If you keep .shaders and .azsl in the Shaders section, and .materialtype and .material under Assets, it should be picked up by the AP.  Try it and once done, leave these files there and point to them.

Reply:
I was just following along with the instructions that were already there. The instructions said to copy the .shader, .azsl, and .material but the .materialtype was missing. All I did was add .materialtype to the list.